ios/FluentUI.Demo/FluentUI.Demo/Demos/SideTabBarDemoController.swift

```diff
@@ -184,14 +184,14 @@ class SideTabBarDemoController: DemoController {
     }
 
     private func showAvatarView(_ show: Bool) {
-        var avatarView: AvatarLegacyView?
+        var avatar: MSFAvatar?
         if let image = UIImage(named: "avatar_kat_larsson"), show {
-            avatarView = AvatarLegacyView(avatarSize: .medium, withBorder: false, style: .circle, preferredFallbackImageStyle: .onAccentFilled)
-            avatarView?.setup(primaryText: "Kat Larson", secondaryText: "", image: image)
-            avatarView?.hasPointerInteraction = true
```

Comment:
If we added pointer interaction, do we want to include this too?

Reply:
Thanks for catching it. I'll set it for the side tab bar too.